docs/testing/cli_testing_frameworks.md

```diff
@@ -4,22 +4,22 @@ title: Stacks Testing Frameworks
 sidebar_label: Stacks Testing Frameworks
 ---
 
-Building a testing frameworks from scratch requires a lot of boilerplate code. It is also hard to know how to get started without examples.
+Building a testing framework from scratch requires a lot of boilerplate code. It is also hard to know how to get started without examples.
```

Comment:
Building testing frameworks from scratch requires a lot of boilerplate code. It is also hard to know how to get started without examples.